src/back/spv/instructions.rs

```diff
@@ -604,12 +604,41 @@ pub(super) fn instruction_binary(
 // Control-Flow Instructions
 //
 
+pub(super) fn instruction_selection_merge(
```

Comment:
Yes I did actually consider this at first. However there was and is at least one case I can remember that doesn't work, without prefixing it. That is `OpReturn`. If we want to change it, it will be called `return`, which is a reserved keyword.

Other than that, at the moment there are no keywords (at the moment) that conflict with Rust.

Reply:
ok, but that's just naming. We can call it `ret()` fwiw, it's fine. It's all internal API anyway.

Reply:
Yes we can indeed, I can do this in a follow up issue. Let me add it to my todo list, so I can create the issue at a later time.

Reply:
Awesome, thank you! As long as it's on our radar, all is well :)